campaign.html

```diff
@@ -25,6 +25,16 @@
 </div>
 
 <div class="margin-top">
+  <p class="stats">
```

Comment:
For consistency's sake, could you remove the class from this paragraph? It's not needed.